app/forms/schools/year2020_form.rb

```diff
@@ -5,28 +5,18 @@ class Year2020Form
     include ActiveModel::Model
     include ActiveModel::Serialization
 
-    attr_accessor :school_id, :induction_programme_choice, :core_induction_programme_id, :full_name, :email, :participants
-
-    validates :induction_programme_choice, presence: true, on: :choose_induction_programme
+    attr_accessor :school_id, :core_induction_programme_id, :full_name, :email, :participants
     validates :core_induction_programme_id, presence: true, on: :choose_cip
 
     validates :full_name, presence: true, on: %i[create_teacher update_teacher]
-    # TODO: unique emails
     validates :email,
               presence: true,
               notify_email: { allow_blank: true },
               on: %i[create_teacher update_teacher]
+    validate :email_is_not_in_use
```

Comment:
Does this need an `on: %i[create_teacher update_teacher]`?

Reply:
It looks like if there is no email, the validation will still pass as it will try to find a user with an email of nil, which is validated against. But yes we can limit this to only those actions to make it more efficient.